src/wp-content/themes/twentyfourteen/functions.php

```diff
@@ -315,7 +315,7 @@ function twentyfourteen_widgets_init() {
 	 * @since Twenty Fourteen 1.0
 	 * @since Twenty Fourteen 3.6 Replaced Google URL with self-hosted fonts.
 	 *
-	 * @return string
+	 * @return string Fonts URL for the theme.
```

Comment:
Good catch that the description was missing in Twenty Fourteen, but I think the description in `twentytwelve_get_font_url()` and `twentythirteen_fonts_url()` would fit better.

```suggestion
	 * @return string Font stylesheet or empty string if disabled.
```

Reply:
How about “Font stylesheet _URL_”?

Reply:
Then I would want "Font stylesheet URL" for T12, T13 and T14 :)

Reply:
I noticed that Twenty Fourteen, Twenty Fifteen, Twenty Sixteen, and Twenty Seventeen all use the `@return string Fonts URL for the theme.` description. Should I update: `@return string Font stylesheet or empty string if disabled. `for consistency?

Reply:
Maybe it should be the same for all six themes from Twenty Twelve to Twenty Seventeen:
`Font stylesheet URL or empty string if disabled.`

I wanted to say something like "if _all fonts are_ disabled" for Twenty Fifteen and Twenty Sixteen because locales can turn off fonts individually, but the same would fit for Twenty Thirteen and that was not any different than the themes with only one hosted font.

Reply:
Thanks for the clarification! I agree using **“Font stylesheet URL or empty string if disabled”** consistently across Twenty Twelve to Twenty Seventeen makes sense. I’ll update all six themes accordingly.